src/main/fc/runtime_config.c

```diff
@@ -29,10 +29,26 @@ uint16_t flightModeFlags = 0;
 
 static uint32_t enabledSensors = 0;
 
+// Must be shorter than OSD_WARNINGS_MAX_SIZE (11) to be displayed fully in OSD
```

Comment:
Is there a simple way that we can assert this at compile time somehow?

Reply:
A way: Yes. Simple: Not that I know of. Also, n.b. that in case a string is longer, it will just be truncated, so the only negative impact is cosmetic.